integration/experiment/gpu_frequency_sweep/gen_gpu_activity_constconfig_recommendation.py

```diff
@@ -89,5 +93,12 @@ def main(full_df):
         sys.exit(1)
 
     output = main(df)
-    sys.stdout.write("AGENT POLICY:\n")
-    sys.stdout.write(json.dumps(output) + "\n")
+
+    root_dir = os.getenv('GEOPM_SOURCE')
+    schema_file = root_dir + "/service/json_schemas/const_config_io.schema.json"
+    with open(schema_file, "r") as f:
+        schema = json.load(f)
+
+    jsonschema.validate(output, schema=schema)
+
+    sys.stdout.write(json.dumps(output, indent=4) + "\n")
```

Comment:
Is the idea to give the user the flexibility to decide what to do with the output (e.g. pipe it to a file)?

I'm thinking sooner or later (perhaps sooner) we'll need to be able to append data to an existing config file. Either because we have separate scripts that generate different sets of configuration parameters (like now) or because a future node characterization tool might provide the flexibility regarding what to characterize, I can see how the file could end up being populated incrementally. Should we incorporate that capability now or do you think the use case for now will be to generate a config file with just the configuration parameters for the agent of interest?